__tests__/app.test.js

```diff
@@ -131,3 +131,63 @@ describe("Get 200: /api/reviews", () => {
       });
   });
 });
+
+describe("/api/reviews/:review_id/comments", () => {
+  test("Get 200: comments should be served with the most recent comments first", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments).toHaveLength(3);
+        expect(comments).toBeInstanceOf(Array);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            review_id: expect.any(Number),
```

Comment:
as these are comments for a specific review, your test will be more robust if each comment is tested for the expected `review_id`, rather than any ol' number